### DISCUSSION TITLE
Order is not cancellable offchain since it does not use SignedZone

### DISCUSSION THREAD

**Discussion by pixelpawnshop** [ProjectOpenSea/opensea-js#1514] [Q&A]
Hello there,

I wanted to try out the cancel order api endpoint for a offer I created. I get this error message though

`Order is not cancellable offchain since it does not use SignedZone`.

I get that if my offer does not provide the SignedZone it only can be cancelled onchain. Im looking for a offchain solution here though. Im currently using the opensea SDK to create my offers. When going through the sdk docs for createOffer() I can't find any information about where to set a SignedZone. Can anyone help on this issue? Thanks.

**Comment by ryanio** [comment]
ah you're right the `offerProtectionEnabled` param is only available on `createCollectionOffer()`

If you want to help open a PR to add param `zone` to `createOffer()`, then we can also add a constant for `export const SIGNED_ZONE = "0x000056f7000000ece9003ca63978907a00ffd100"` and passing that to `createOffer()` should work.
